taskqueue/gcloud/aio/taskqueue/taskqueue.py

```diff
@@ -51,9 +51,17 @@ def __init__(self, project, service_file, taskqueue, location=LOCATION,
         resp.raise_for_status()
         return data
 
+
+class PullQueue(BaseQueue):
```

Comment:
Could you split these files up? `pullqueue.py` and `pushqueue.py` (and the tests should follow the same divide). You can throw the base class and any common code into something like `base.py` or `common.py`.

Reply:
Note: I moved the new `PushQueue` name to `pushqueue.py` in my deprecation release, which should help pave the way for this.